NewForce-Cohort-4/tabloid-full-stack-coral
# JN List Categories

# Description

Added a GetAll method in `CategoryRepository.cs` to access database, updated interface and controller. Updated `CategoryProvider.js` with GetAll function. Added `Category.js` and `CategoryList.js` to render new views.

Fixes Trello Ticket [#9 View All Categories](https://trello.com/c/tYO2883b/9-9-view-all-categories)

## Type of change


# How Has This Been Tested?

- In Terminal/ git-bash `cd` to the project repository
- Run the command: `git fetch --all && git checkout jn-list-post`
- Open the .sln file in Visual Studio 2019 and start the project
- In Terminal/ git-bash, navigate to the client directory: `cd ./Tabloid/client/`
- Run `npm start` in the client directory
- Login with a user who has previously authored a post, e.g.: `foo@bar.com`.
- Click `Category Management`
- Confirm a list of categories appears in the browser

# Checklist:
